app/src/main/java/com/github/bgabriel998/softwaredevproject/CompassView.java

```diff
@@ -60,45 +60,60 @@ public CompassView(Context context, AttributeSet attrs){
     }
 
     /**
-     * Initializes all needed widgets like paint variables
+     * Initializes all needed widgets for the compass like paint variables
      */
     private void widgetInit(){
         //Initialize colors
-        textColor = R.color.Black;
-        lineColor = R.color.Black;
+        compassColor = R.color.Black;
 
         //Initialize fonts
         float screenDensity = getResources().getDisplayMetrics().scaledDensity;
-        mainTextSize = (int) (15 * screenDensity);
-        int secondaryTextSize = (int) (12 * screenDensity);
+        mainTextSize = (int) (20 * screenDensity);
 
         //Initialize paints
-        //Paint used for the mian text heading (N, E, S, W)
-        mainTextPaint = new Paint(Paint.ANTI_ALIAS_FLAG);
-        mainTextPaint.setTextAlign(Paint.Align.CENTER);
-        mainTextPaint.setColor(textColor);
-        mainTextPaint.setTextSize(mainTextSize);
+        //Paint used for the main text heading (N, E, S, W)
+        mainTextPaint = configureTextPaint(mainTextSize);
 
         //Paint used for the secondary text heading (NE, SE, SW, NW)
-        secondaryTextPaint = new Paint(Paint.ANTI_ALIAS_FLAG);
-        secondaryTextPaint.setTextAlign(Paint.Align.CENTER);
-        secondaryTextPaint.setColor(textColor);
-        secondaryTextPaint.setTextSize(secondaryTextSize);
+        secondaryTextPaint = configureTextPaint(15*screenDensity);
 
         //Paint used for the main lines (0°, 90°, 180°, ...)
-        mainLinePaint = new Paint(Paint.ANTI_ALIAS_FLAG);
-        mainLinePaint.setStrokeWidth(8f);
-        mainLinePaint.setColor(lineColor);
+        mainLinePaint = configureLinePaint(5*screenDensity);
 
         //Paint used for the secondary lines (45°, 135°, 225°, ...)
-        secondaryLinePaint = new Paint(Paint.ANTI_ALIAS_FLAG);
-        secondaryLinePaint.setStrokeWidth(6f);
-        secondaryLinePaint.setColor(lineColor);
+        secondaryLinePaint = configureLinePaint(3*screenDensity);
 
         //Paint used for the terciary lines (15°, 30°, 60°, 75°, 105°, ...)
-        terciaryLinePaint = new Paint(Paint.ANTI_ALIAS_FLAG);
-        terciaryLinePaint.setStrokeWidth(3f);
-        terciaryLinePaint.setColor(lineColor);
+        terciaryLinePaint = configureLinePaint(2*screenDensity);
+    }
+
+    /**
+     * Method to create the line paints for the compass
+     * @param strokeWidth width of the lines
+     * @return configured paint
+     */
+    private Paint configureLinePaint(float strokeWidth){
+        int MAX_ALPHA = 255;
```

Comment:
Make `MAX_ALPHA` class global and static final (since it is also used in following method `configureTextPaint`